src/main/protocol.ts

```diff
@@ -66,6 +67,8 @@ const handlePotentialProtocolLaunch = (url: string) => {
     default:
       return;
   }
+  const win = getOrCreateMainWindow();
+  win.show();
```

Comment:
```suggestion

  // Focus the window in case it is in the background
  getOrCreateMainWindow.focus();
```

Reply:
@dsanders11 Thanks for the feedback. Yes the focus method is more suitable here and also more readable.

I will sure write tests for this change.

Reply:
So I double checked and we actually [mock out `BrowserWindow`](https://github.com/electron/fiddle/blob/main/tests/mocks/browser-window.ts) for tests, so the test can just confirm that `focus` is called. 🙂